plugins/AttributeTable.jsx

```diff
@@ -545,7 +554,12 @@ class AttributeTable extends React.Component {
             features: this.state.filteredSortedFeatures.filter(feature => this.state.selectedFeatures[feature.id] === true && feature.geometry)
         };
         if (!isEmpty(collection.features)) {
-            this.props.zoomToExtent(geojsonBbox(collection), this.props.mapCrs);
+            if (collection.features.length == 1 && collection.features[0].geometry.type === "Point"){
```

Comment:
Does this work with a geometryless feature?

Reply:
I'm afraid not. But I could just use the zoomToExtent if it's a geometryless feature, right?

Reply:
Ah right the collection anyway only includes features with geometries, so all good.